mixins/dynamicComponentHeight.ts

```diff
@@ -0,0 +1,18 @@
+import { isServer } from '@vue-storefront/core/helpers';
+
+export default {
+  data () {
+    return {
+      dynamicComponentHeight: 0
+    }
+  },
+  beforeMount () {
```

Comment:
listener will be added for each component that will use this mixin, because each component will call beforeMount

Maybe instead of mixin add something like this https://github.com/DivanteLtd/vue-storefront/blob/master/core/helpers/index.ts#L193-L203 to helpers https://github.com/DivanteLtd/vsf-default/blob/master/helpers/index.ts ?

```
export const windowHelper = Vue.observable({
  height: 0
})
!isServer && window.addEventListener('resize', () => { windowHelper.height = window.innerHeight })
```

Reply:
Awesome! I will do it!